### ISSUE TITLE
Cloud-Ex06-Detailed issues with screenshot

### ISSUE THREAD

**Issue by morexwell** [NorhanKhaled/SkillsAcademy-CAD#83]
I have search all over the internet, I could not find solution for this.. I am uncertain if its update issues or bug issues because most people on windows machine seems to be facing the same.

Command terminal on windows machine..
1.At this point below you see the message..
**”cluster-config is not a registered command”** see screenshot
![cluster-config is not a registered command](https://user-images.githubusercontent.com/11416140/81910505-e9422480-9580-11ea-9e5b-59d9a0f89c60.JPG)

2.There was no notification on the terminal to either “Set the KUBECONFIG environment variable“  **Or**  “export environment variables to start using kubernetes”

3.I Verified that the kubectl commands run properly with cluster by checking the Kubernetes CLI server version: with this command: 
kubectl version --short
![kubectl version](https://user-images.githubusercontent.com/11416140/81910790-3faf6300-9581-11ea-8227-6610806fb0b4.JPG)

4. i proceeded to running “ibmcloud dev create ” command.
At the end of the process, the below is printed out.

saved successfully to current directory

![hello-student saved successfully locally](https://user-images.githubusercontent.com/11416140/81910901-666d9980-9581-11ea-9eef-d459dedaf9d1.JPG)

5. When I run “ kubectl apply -f 01hello-student-deployment.yaml “

'kubectl' is not recognized as an internal or external command,
operable program or batch file.

See below: 

![kubectl is not recognized as an internal or external command](https://user-images.githubusercontent.com/11416140/81910997-843afe80-9581-11ea-94f0-df918bae58c4.JPG)

Any assistance or help  why this error is command is displaying will be appreciated.

Thank you

**Comment by NorhanKhaled** [comment]
1. the command should be ibmcloud ks cluster config --cluster <cluster name >
2. when you run the command in point 1 correctly it will appear , you will need to copy it and paste 
5. I think you missed a step in configuring kubectl on your machine , in the Wellington Ogbemudia location can you see a file called kubectl.exe ??

**Comment by NorhanKhaled** [comment]
I think you didn't apply this note , if you find it on Wellington Ogbemudia path only 
![image](https://user-images.githubusercontent.com/21022185/81928351-cac64280-95ed-11ea-9026-c60f75933331.png)

**Comment by morexwell** [comment]
> I think you didn't apply this note , if you find it on Wellington Ogbemudia path only
> ![image](https://user-images.githubusercontent.com/21022185/81928351-cac64280-95ed-11ea-9026-c60f75933331.png)

how i do add to path? please give an example..  thank you

**Comment by NorhanKhaled** [comment]
are you using windows ? 7 or 10 ?? and I will give you detailed steps

**Comment by morexwell** [comment]
> are you using windows ? 7 or 10 ?? and I will give you detailed steps

window 7 .... thank you

**Comment by morexwell** [comment]
> 1. the command should be ibmcloud ks cluster config --cluster
> 2. when you run the command in point 1 correctly it will appear , you will need to copy it and paste
> 3. I think you missed a step in configuring kubectl on your machine , in the Wellington Ogbemudia location can you see a file called kubectl.exe ??

yes. there is kubectl.exe in that location as : C:\Users\Wellington Ogbemudia\kubectl.exe

**Comment by NorhanKhaled** [comment]
use those steps here https://support.shotgunsoftware.com/hc/en-us/articles/114094235653-Setting-global-environment-variables-on-Windows 
and add new environment variable kubectl and the path is C:\Users\Wellington Ogbemudia\
